src/test/java/com/example/rets_api/repository/BaseTests.java

```diff
@@ -0,0 +1,126 @@
+package com.example.rets_api.repository;
+
+import com.example.rets_api.entity.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class BaseTests {
```

Comment:
Please put this class in utils package and change the name to something related to comparison utils....

Reply:
done - I have put it under utils and changed its name to CompareEntitiesTest